### PR TITLE
Add beta banner and back link

### PR DESCRIPTION
**Pull request by hursey013**
Fixes https://github.com/usdoj-crt/beta.ADA.gov-Project-Management/issues/183

Screen Shot 2021-06-04 at 11 08 51 AM

Screen Shot 2021-06-04 at 11 09 00 AM

### CODE REVIEW

**Comment by c2nelson** [comment]
Should we change see to find?

File: _includes/banner-beta.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  <div
+    class="usa-banner__inner border-top border-primary-lightest padding-top-1"
+  >
+    <div class="grid-col-auto">
+      <span class="usa-tag bg-warning height-1 width-2 display-block"></span>
+    </div>
+    <div class="grid-col-fill">
+      <p class="usa-banner__header-text">
+        <strong>This is a BETA site:</strong> an early, in-progress version of
+        an improved ADA.gov. If you don't see what you're looking for,
```
Comment (hursey013) [inline _includes/banner-beta.html:11]:
```suggestion
        an improved ADA.gov. If you don't find what you're looking for,
```

**Comment by hursey013** [comment]
@c2nelson I switched the `see` to `find` so going to merge this in now!

**Comment by c2nelson** [comment]
thank you, brian.  sorry i didn't get to this.

**Pull request merged**